### DISCUSSION TITLE
Cant upgrade to 2.0 version

### DISCUSSION THREAD

**Discussion by k3ybladewielder** [apache/airflow#14427] [General]
I'm trying to upgrade to version 2.0, I already installed apache-airflow-upgrade-check but when running airflow upgrade_check asks to install apache-airflow-upgrade-check

**Comment by turbaszek** [comment]
@k3ybladewielder what Airflow version do you use?

**Reply by k3ybladewielder** [reply]
1.10.13. Im trying to upgrade to bride to upgrade to 2.0

**Reply by turbaszek** [reply]
Please try installing 1.10.14 and then installing upgrade check - this should help. CC @ephraimbuddy

**Reply by k3ybladewielder** [reply]
I'm not able to install 1.14 because you need apache-airflow-upgrade-check, even if already installed, ask to install

**Reply by ephraimbuddy** [reply]
I'm not sure how you're trying to install this, do you mean 1.10.14? you wrote 1.14. Also, can you show the commands you used?

**Reply by turbaszek** [reply]
To install 1.10.14 you don't need upgrade check - 1.10.14 does not break compatibility 👌

**Reply by k3ybladewielder** [reply]
> I'm not sure how you're trying to install this, do you mean 1.10.14? you wrote 1.14. Also, can you show the commands you used?

https://airflow.apache.org/docs/apache-airflow/stable/upgrading-to-2.html

**Reply by turbaszek** [reply]
@k3ybladewielder have you been able to solve the problem?

**Reply by k3ybladewielder** [reply]
> @k3ybladewielder have you been able to solve the problem?

No
